storage/engine/engine.go

```diff
@@ -39,7 +39,7 @@ type Iterator interface {
 	// is <= the provided key.
 	SeekReverse(key MVCCKey)
 	// Valid returns true if the iterator is currently valid. An
-	// iterator which hasn't been seeked or has gone past the end of the
+	// iterator which hasn't been sought or has gone past the end of the
```

Comment:
seek is a method name, so whether or not this is a real word doesn't matter. How about `seek()`ed?